spot_rl_experiments/configs/config.yaml

```diff
@@ -1,16 +1,15 @@
 WEIGHTS:
   # Regular Nav
-  NAV: "weights/CUTOUT_WT_True_SD_200_ckpt.99.pvp.pth"
-
+  NAV: "weights/torchscript/CUTOUT_WT_True_SD_200_ckpt.99.pvp_combined_net.torchscript" #"weights/CUTOUT_WT_True_SD_200_ckpt.99.pvp.pth"
```

Comment:
Could you please clean up the "#" sign?

Reply:
That comment was kept if anyone wants to use previous pytorch weights, they can just uncomment & use pytorch weights without any changes in the code.
If you want to me to remove these comments, I can remove those.